### PR TITLE
Kellyyangsong/net8 ga

### PR DESCRIPTION
**Pull request by kellyyangsong**

### CODE REVIEW

**Review by ciaozhang — APPROVED** [review]

File: .github/workflows/dotnetcore.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ jobs:
     - name: Setup .NET 8.0.x
       uses: actions/setup-dotnet@v2.1.0
       with:
-        dotnet-version: 8.0.100-rc.1.23463.5
+        dotnet-version: 8.0.100
```
Comment (keegan-caruso) [inline .github/workflows/dotnetcore.yml:42]: maybe 8.0.x? Not sure. we float patch for 7 but use a specific version for 6.

Comment (jennyf19) [reply .github/workflows/dotnetcore.yml:42]: 8.0.x should work. I think on 6 we had to target a specific version at one time.

Comment (jennyf19) [reply .github/workflows/dotnetcore.yml:42]: nit: could you start from a new branch instead of merging w/main from a previous branch?

Comment (kellyyangsong) [reply .github/workflows/dotnetcore.yml]: donezo

Comment (kellyyangsong) [reply .github/workflows/dotnetcore.yml]: > nit: could you start from a new branch instead of merging w/main from a previous branch?

yes 😬

**Review by ciaozhang — APPROVED** [review]

File: build/template-install-dependencies.yaml [hunk 2]
```diff
@@ -22,7 +22,6 @@ steps:
   displayName: 'Use .Net Core SDK 8'
   inputs:
     version: 8.x  
-    includePreviewVersions: true
   condition: eq(variables['TargetNet8'], 'True')
```
Comment (jennyf19) [inline build/template-install-dependencies.yaml:25]: this needs to be removed as well.

Comment (kellyyangsong) [reply build/template-install-dependencies.yaml:25]: `includePreviewVersions: true` was removed unless you're talking about a different line?

Comment (jennyf19) [reply build/template-install-dependencies.yaml:25]: yes line 25 or 26 - the one with `condition: ...`

Comment (kellyyangsong) [reply build/template-install-dependencies.yaml]: done :)

**Review by jennyf19 — APPROVED** [review]
:shipit: 

next time, start from a clean `git pull` from main and start a new branch.

**Pull request merged**